docs/aws_sso.md

```diff
@@ -0,0 +1,56 @@
+If you are working with AWS, there are two things you should avoid at all costs:
+
+* Use account’s root credentials for day-to-day work
+* Provision individual users in your account and generate credentials for them
+
+AWS Single Sign On (SSO) with Athenz allows you to avoid both. Because an Athenz
+domain is directly related to an AWS account, the same authorization mechanism
+can be leveraged for SSO to the AWS console, via SAML-based integration. The
+Identity Provider service validates the user and then uses Athenz
+to determine the AWS role(s) the user can assume. This set of roles also indirectly
+identifies all the accounts the user can access from the console. In its simplest
+form, the single (potentially large) list can be presented to AWS in a SAML
+document, and AWS allows selection of the account/role to use.
+
+## Onboard to AWS
+
+To register an AWS Account with a domain, run the following command:
+```
+zms-cli -d <domain-name> set-aws-account <aws-account-id>
+```
+
+In the Athenz UI, when you select your domain,
+you should see your onboarded AWS account number.
+
+Once your AWS account creation has successfully completed, it should create the
+following 3 required roles for AWS Console SSO with their corresponding policies:
+
+* aws.fed.admin.user - Members of this role get admin privileges for your AWS
+account. This is ROOT for AWS accounts. No one should be assigned this role,
+unless they are performing ROOT functions
+* aws.fed.power.user - This role restricts access to IAM functions, VPC core
+functions, etc.  Your day to day tasks should use this role.
+* aws.fed.read.only - ReadOnly Access to the AWS account.
+
+![AWS Federated roles in Athenz](images/aws_fed_roles.png)
+
+## Add members to AWS SSO roles
```

Comment:
Removed entire section on "Access AWS Console". Original text:
## Access AWS Console

 Athenz is IdP for AWS console Access. It uses Okta multi factor authentication
 and is accessible to Verizon Media as a whole.

 For non-admin accounts use the following link:
 [https://ui.athenz.ouroath.com/aws/sso/dev](https://ui.athenz.ouroath.com/aws/sso/dev)
 ([yo/aws-sso](https://ui.athenz.ouroath.com/aws/sso/dev))

 ![AWS sso](images/aws_non_admin_login.png)

 For admin accounts this is the link:
 [https://ui.athenz.ouroath.com/aws/sso/admin](https://ui.athenz.ouroath.com/aws/sso/admin)
 ([yo/aws-ssoadmin](https://ui.athenz.ouroath.com/aws/sso/admin))

 ![AWS sso](images/aws_admin_login.png)"